src/system/random/basic.lean

```diff
@@ -250,6 +250,15 @@ instance nat_bounded_random : bounded_random ℕ :=
      pure ⟨z.val + x, nat.le_add_left _ _,
        by rw ← nat.le_sub_right_iff_add_le hxy; apply le_of_succ_le_succ z.is_lt⟩ }
 
+instance int_bounded_random : bounded_random ℤ :=
```

Comment:
Could we have a doc-string describing approximately how these `int`s are produced?